Chapters/MorphicAthens/MorphicAthens.md

```diff
@@ -3,210 +3,190 @@
 status: Did a first pass on the material of renaud
 status: spellchecked
 
-This chapter has been originally written by renaud villemeur. We thank him for this contribution. It shows how you can integrate vector graphic drawing within Spec components.
+This chapter has been originally written by Renaud Villemeur. We thank him for his contribution. It shows how you can integrate vector graphic drawing within Spec components.
```

Comment:
Renaud de Villemeur

Reply:
Fixed in https://github.com/SquareBracketAssociates/BuildingApplicationWithSpec2/pull/62